airbyte-integrations/connectors/source-goldcast/source_goldcast/schemas/event_agenda.json

```diff
@@ -0,0 +1,191 @@
+{
```

Comment:
I would prefer to inline schema in the yaml manifest, but again, can come later. Esp if you're interested in making more sources!

Reply:
@natikgadzhi , Its done as well ! Would you like me to delete these json schemas ? Is it not required ?